### ISSUE TITLE
When a certain module's path is substring of anther module's path，mvn generate throws exception

### ISSUE THREAD

**Issue by cnzhujie** [EvoSuite/evosuite#310]
My project structure as follows：
```
.
├── README
├── contra-apitest
├── contra-apitest-agent-client
```
Module **contra-apitest** dependent on module **contra-apitest-agent-client**

When I run `mvn org.evosuite.plugins:evosuite-maven-plugin:1.0.6:generate xxx` error happens:
```
[INFO] [MASTER] 17:13:40.479 [main] ERROR EvoSuite - Fatal crash on main EvoSuite process. Class  using seed 1601025220475. Configuration id : null
[INFO] java.lang.IllegalArgumentException: Classpath element does not exist on disk at: /Users/mtdp/IdeaProjects/contra/contra-apitest/target/classes:/Users/mtdp/IdeaProjects/contra/contra-apitest-agent-client/target/classes
[INFO]  at org.evosuite.classpath.ClassPathHandler.checkIfValidClasspathEntry(ClassPathHandler.java:211) ~[evosuite-master-1.0.6.jar:1.0.6]
[INFO]  at org.evosuite.classpath.ClassPathHandler.addElementToTargetProjectClassPath(ClassPathHandler.java:188) ~[evosuite-master-1.0.6.jar:1.0.6]
[INFO]  at org.evosuite.CommandLineParameters.handleClassPath(CommandLineParameters.java:238) ~[evosuite-master-1.0.6.jar:1.0.6]
[INFO]  at org.evosuite.EvoSuite.parseCommandLine(EvoSuite.java:184) ~[evosuite-master-1.0.6.jar:1.0.6]
[INFO]  at org.evosuite.EvoSuite.main(EvoSuite.java:302) ~[evosuite-master-1.0.6.jar:1.0.6]
[ERROR] Error in EvoSuite
[INFO] Stopping spawn process manager
```

This happens because `/Users/mtdp/IdeaProjects/contra/contra-apitest-agent-client/target/classes` startsWith `/Users/mtdp/IdeaProjects/contra/contra-apitest`

Can we just use `project.getBuild().getOutputDirectory()` alternatively?
![image](https://user-images.githubusercontent.com/4935935/94251511-35509d00-ff55-11ea-95ce-632f14cf6735.png)

**Comment by gofraser** [comment]
I haven't managed to reproduce this, even using exactly the same submodule names as in your example. Are there any dependencies between the submodules? Could you maybe provide a minimal example I could use to debug this?

**Comment by gofraser** [comment]
Nevermind, managed to reproduce it. I'll look into it.

**Comment by gofraser** [comment]
The suggested fix seems fine; this should be fixed in 91089126e59c188423fc3038b8f38c5013f693ef

**Comment by cnzhujie** [comment]
> The suggested fix seems fine; this should be fixed in [9108912](https://github.com/EvoSuite/evosuite/commit/91089126e59c188423fc3038b8f38c5013f693ef)

I have tested with my project, it works.
